dockerfile-generator/dockerfile-gen-services/src/main/java/io/hyscale/dockerfile/gen/services/exception/DockerfileErrorCodes.java

```diff
@@ -15,28 +15,40 @@
  */
 package io.hyscale.dockerfile.gen.services.exception;
 
-import io.hyscale.commons.exception.HyscaleErrorCode;
+import io.hyscale.commons.exception.HyscaleError;
+import io.hyscale.commons.exception.HyscaleErrorGroup;
 
-public enum DockerfileErrorCodes implements HyscaleErrorCode {
-    FAILED_TO_GENERATE_DOCKERFILE("Failed to generate Dockerfile"),
+public enum DockerfileErrorCodes implements HyscaleError {
+    FAILED_TO_GENERATE_DOCKERFILE("Failed to generate Dockerfile",HyscaleErrorGroup.DOCKER_FILE_GENERATION),
     FAILED_TO_COPY_ARTIFACT("Failed to copy artifacts"),
     FAILED_TO_CREATE_SCRIPT("Failed to create script"),
     FAILED_TO_PERSIST_DOCKERFILE("Failed to persist Dockerfile"),
-    BUILD_SPEC_REQUIRED("BuildSpec Required"),
-    DOCKERFILE_OR_BUILDSPEC_REQUIRED("Either Dockerfile or BuildSpec is required to build an image"),
-    FAILED_TO_PROCESS_DOCKERFILE_GENERATION("Failed to process Dockerfile Generation"),
+    BUILD_SPEC_REQUIRED("BuildSpec Required",HyscaleErrorGroup. DOCKER_FILE_GENERATION),
+    DOCKERFILE_OR_BUILDSPEC_REQUIRED("Either Dockerfile or BuildSpec is required to build an image",HyscaleErrorGroup.DOCKER_FILE_GENERATION),
+    FAILED_TO_PROCESS_DOCKERFILE_GENERATION("Failed to process Dockerfile Generation",HyscaleErrorGroup.DOCKER_FILE_GENERATION),
     ARTIFACTS_FOUND_INVALID_IN_SERVICE_SPEC("Artifacts found to be invalid in the given service spec"),
     CANNOT_RESOLVE_STACK_IMAGE("Cannot resolve stack image");
```

Comment:
Same docker file generation error group

Reply:
As per our initial discussion, I did not assigned the error groups for those being not used anywhere. So I ignored the above. Please let me know whether to assign those or not?

Reply:
Completed in the latest commit.